modules/manage/pages/iceberg/iceberg-topics-aws-glue.adoc

```diff
@@ -40,21 +40,65 @@ endif::[]
 === Nested partition spec support
 
 AWS Glue does not support partitioning on nested fields. If Redpanda detects that
-the default partitioning `(hour(redpanda.timestamp))` is in use, it will instead apply an empty partition spec `()`, which means the table will not be partitioned.
+the default partitioning `(hour(redpanda.timestamp))` based on the record metadata is in use, it will instead apply an empty partition spec `()`, which means the table will not be partitioned.
 
-If you want to use partitioning, you must specify a custom partition specification using your own partition columns (columns that are not nested).
+If you want to use partitioning, you must implement custom partitioning using your own partition columns (columns that are not nested).
+
+[NOTE]
+====
+In Redpanda versions 25.2.1, and 25.1.9 and earlier, an empty partition spec () can cause a known issue that prevents certain engines like Amazon Redshift from successfully querying the table. To resolve this issue, specify custom partitioning, or upgrade Redpanda to versions 25.2.2 or 25.1.10 and later.
```

Comment:
```suggestion
In Redpanda versions 25.2.1, 25.1.9, and earlier, an empty partition spec () can cause a known issue that prevents certain engines like Amazon Redshift from successfully querying the table. To resolve this issue, specify custom partitioning or upgrade Redpanda to versions 25.2.2, 25.1.10, or later.
```
This is confusing. Is there a version in between 25.1.10 and 25.2.1? Seems possible to upgrade from 25.1.9 or an earlier version to 25.2.1 and find yourself in the same problem. To make it safe and simple, can we just say "In version 25.2.1 and earlier... specify custom partitioning or upgrade Redpanda to version 25.2.2 or later."